learning/katas/go/Introduction/Hello Beam/Hello Beam/test/task_test.go

```diff
@@ -0,0 +1,46 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"io/ioutil"
+	"task"
+	"testing"
+)
+
+func TestTask(t *testing.T) {
```

Comment:
I propose to avoid testing using output file to simplify (refer to task.go review comment).
```go
func TestTask(t *testing.T) {
	ctx := context.Background()

	p, s := beam.NewPipelineWithRoot()

	helloBeam := main.HelloBeam(s)

	passert.Equals(s, helloBeam, "Hello Beam")

	if err := beamx.Run(ctx, p); err != nil {
		log.Exitf(ctx, "Failed to execute job: %v", err)
	}
}
```